libusbhid_test.pp

```diff
@@ -18,7 +18,7 @@
   cthreads,
   {$ENDIF}{$ENDIF}
 
-  keyboard,
+  crt,

```

Comment:
I am very wary of using the crt unit. I had only trouble with it in the past, memory leaks in multithreaded applications because of it. In my experience the keyboard unit it a much better choice. I have no immediate way to test on windows 10 but, I realized that the keyboard needs to be initialized and finalized at the beginning and at the end of the program (InitKeyboard() and DoneKeyboard(). I updated my example, maybe you can give it another try on windows 10.

Reply:
OK, will test it now

Reply:
Keyboard unit is working fine now on Windows 10